libs/features/personalization/personalization.js

```diff
@@ -180,15 +180,16 @@ function normalizeKeys(obj) {
 function handleCommands(commands, manifestId, rootEl = document) {
   commands.forEach((cmd) => {
     if (VALID_COMMANDS.includes(cmd.action)) {
-      let selectorEl = rootEl.querySelector(cmd.selector);
-
-      if (!selectorEl) return;
-
-      if (selectorEl.classList[0] === 'section-metadata') {
-        selectorEl = selectorEl.parentElement || selectorEl;
+      try {
+        let selectorEl = rootEl.querySelector(cmd.selector);
```

Comment:
Two points:
- Should we have the try/catch only around the querySelector? 
- I've seen this happen in two other places in the codebase. I wonder if a helper `safeQuerySelector` that just wraps `querySelector/querySelectorAll` in a try/catch would make sense

Reply:
good point, I see quite some other usages. but if it's ok with everyone, I'd like to do it in the next PR.
need to analyze which of those values are Author-produced and which are hardcoded in code (and can be considered safe?).
<img width="942" alt="image" src="https://github.com/adobecom/milo/assets/6617234/743b651d-e1a2-4797-a037-74a600599321">